docs/src/man/getting_started.md

```diff
@@ -891,45 +980,45 @@ julia> x = [3, 1, 2];
 
 julia> df = DataFrame(x=x)
 3×1 DataFrame
-│ Row │ x     │
-│     │ Int64 │
-├─────┼───────┤
-│ 1   │ 3     │
-│ 2   │ 1     │
-│ 3   │ 2     │
+ Row │ x
+     │ Int64
+─────┼───────
+   1 │     3
+   2 │     1
+   3 │     2
 
 julia> sort!(df)
 3×1 DataFrame
-│ Row │ x     │
-│     │ Int64 │
-├─────┼───────┤
-│ 1   │ 1     │
-│ 2   │ 2     │
-│ 3   │ 3     │
+ Row │ x
+     │ Int64
+─────┼───────
+   1 │     1
+   2 │     2
+   3 │     3
 
 julia> x
 3-element Array{Int64,1}:
+ 3
  1
  2
- 3
 
 julia> df.x[1] = 100
 100
 
 julia> df
 3×1 DataFrame
-│ Row │ x     │
-│     │ Int64 │
-├─────┼───────┤
-│ 1   │ 100   │
-│ 2   │ 2     │
-│ 3   │ 3     │
+ Row │ x
+     │ Int64
+─────┼───────
+   1 │   100
+   2 │     2
+   3 │     3
 
 julia> x
```

Comment:
While we're at it, probably better not print `x` here as it's not mutated, which is kind of the opposite of what the text above says. Or we should explain that `DataFrame(x=x)` makes a copy.

Reply:
I think this was the point of the original example. I have added a comment below.